movies-app/src/pages/MovieList.js

```diff
@@ -0,0 +1,63 @@
+import { useCallback, useState } from 'react';
+import '../App.css';
+import { Default as Counter } from '../stories/Counter.stories';
+import { Default as Search } from '../stories/Search.stories';
+import { Default as Genre } from '../stories/Genre.stories';
+import { AddMovieModal } from '../stories/Modal.stories';
+import { GenresList, SortType } from '../constants/constants';
+import MovieDetails from '../components/MovieDetails/MovieDetails';
+import MovieTile from '../components/MovieTile/MovieTile';
+import SortBy from '../components/SortBy/SortBy';
+
+const initialCounterValue = 0;
```

Comment:
why do we need to define the values outside the fn ? we can initialize the values in useState..

Reply:
Moved to constants and used in state